exonum/src/messages/mod.rs

```diff
@@ -55,6 +55,8 @@ pub enum Any {
     Request(RequestMessage),
     /// Transaction.
     Transaction(RawTransaction),
+    /// A bunch of transactions.
+    TransactionsBunch(TransactionsResponse),
```

Comment:
`Bunch` intuitively seems wrong to me; `TransactionsBundle` or `TransactionsBatch` both sound better. @Maria-Nosyk?

Reply:
bunch isn't absolutely wrong but has a slightly different connotation, besides batch is more common for us as I could notice from our docs

Reply:
Personally, I prefer to use "batch" term here.